node/core/pvf/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition.workspace = true
 [[bin]]
 name = "puppet_worker"
 path = "bin/puppet_worker.rs"
+required-features = ["test-utils"]
 
 [dependencies]
```

Comment:
Sorry for the delay on this one. Done this, and added it as an optional dependency instead of dev, from what I understood the feature cannot use dependencies that are in the dev dependencies. Let me know if this is right.

Reply:
I've tried it and you're right, looks good 👍